target/linux/ipq40xx/image/Makefile

```diff
@@ -193,6 +193,32 @@ define Device/compex_wpj428
 endef
 TARGET_DEVICES += compex_wpj428
 
+define Device/dlink_dap-2610
+	$(call Device/FitImageLzma)
+	DEVICE_VENDOR := D-Link
+	DEVICE_MODEL := DAP-2610
+	DEVICE_DTS := qcom-ipq4018-dlink-dap-2610
```

Comment:
Due to 9c7025b7794ae06774632499547f78d827d13499 , DEVICE_DTS is dropped in favor of 

SOC := ipq4018

(you'll need to rename the dts to qcom-ipq4018-dap-2610.dts)